libs/net/dal/Services/ReportInstanceService.cs

```diff
@@ -100,7 +100,10 @@ public override ReportInstance Add(ReportInstance entity)
         {
             ric.Instance = entity;
             ric.InstanceId = entity.Id;
-            this.Context.ReportInstanceContents.Add(ric);
+            if (ric.ContentId == 0)
```

Comment:
Fixed bug when generating a new instance was attempting to add content that already existed.